buf/registry/descriptor/v1/file_descriptor_proto_extension.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.registry.descriptor.v1;
+
+import "buf/registry/priv/extension/v1beta1/extension.proto";
+import "buf/validate/validate.proto";
+
+option go_package = "buf.build/gen/go/bufbuild/registry/protocolbuffers/go/buf/registry/descriptor/v1";
+
+// Additional information about a FileDescriptorProto.
+//
+// FileDescriptoProtos are not extendable, so we choose to sideband this information as opposed
+// to creating a new FileDescriptoProto type, so that the FileDescriptorProtoService can return
```

Comment:
```suggestion
// FileDescriptorProtos are not extendable, so we choose to sideband this information as opposed
// to creating a new FileDescriptorProto type, so that the FileDescriptorProtoService can return
```
drive-by!